python/secure_install/README.md

```diff
@@ -0,0 +1,41 @@
+# Architecture-as-Code Secure Installation
+In order to support portability of Architecture-as-Code, and to provide a secure, repeatable installation method for AaC users, we have created
+this one-stop installation package for each release of the AaC python package.
+
+In this package we provide a couple different methods to install AaC and its dependencies, depending on your needs and environment.
+
+## Hash-Verified PyPI Dependencies
+When we create a release of AaC, we'll include a list of the runtime dependencies for AaC and their hash values. When you install via `pip install --require-hashes -r requirements.txt`
+it will pull the pinned package versions, and verify the package's hash value against the value in the requirements file for data integrity. If you prefer to avoid executing the long pip command, you can run the `install_aac_verified_index` script. The installed artifacts will be sourced from PyPI.
+
+
+### Linux/macOS
+Github's artifact archiver doesn't preserve permissions so you'll have to make the script executable.
```

Comment:
So, that was my initial implementation, but it requires the user to unzip then untar the files. So, either way, the user will have to run multiple commands on a linux system to install AaC. This current implementation allows windows users to avoid that.